assets/components/pageSection/pageSection.scss

```diff
@@ -0,0 +1,13 @@
+.component-page-section__header {
+  @include mq($from: desktop) {
+    display: inline-block;
+    vertical-align: top;
+    width: 220px;
```

Comment:
Is this width value standard? If so, is it worth to add it to the Layout.scss?

Reply:
Hard to know yet, we haven't got a comprehensive style guide yet.